JSTests/wasm/stress/simple-inline-stacktrace.wat

```diff
@@ -0,0 +1,72 @@
+;; wat2wasm --enable-exceptions simple-inline-stacktrace.wat -o simple-inline-stacktrace.wasm --debug-names
```

Comment:
Note: debug names are totally broken ATM, I might fix them in a subsequent patch.